docs/modules/ROOT/pages/how-tos/configuring/prefetching-dependencies.adoc

```diff
@@ -289,6 +294,84 @@ spec:
 * From the {ProductName} *Applications* view, go to *Activity > Latest commits*.
 
+== Enabling prefetch builds for `rpm`
+
+Cachi2 has a dev-preview package manager capable of fetching `rpm` dependencies. This requires the use of a pair of `rpms.in.yaml` and `rpms.lock.yaml` files to be committed to your repository. You write a `rpms.in.yaml` file and the link:https://github.com/konflux-ci/rpm-lockfile-prototype?tab=readme-ov-file#what-is-this[rpm-lockfile-prototype] CLI tool resolves that to produce a `rpms.lock.yaml` file. Cachi2 fetches those specific rpms and enables your build to install them without network access.
+
+.Prerequisites
+* You have an up-to-date installation of link:https://github.com/konflux-ci/rpm-lockfile-prototype?tab=readme-ov-file#installation[rpm-lockfile-prototype].
+
+.Procedure
+
+To prefetch dependencies for a component build, complete the following steps:
+
+. Create a `rpms.in.yaml` file in your git repository, with the following contents:
+
++
+[source,yaml]
+----
+packages: [nethack] <1>
+contentOrigin:
+  repofiles: ["./fedora.repo"] <2>
+----
+<1> The `*packages*` list is the list of packages you want to install in your Container. Only include first-order dependencies here, i.e. the packages you want to install.
+<2> This should be a reference to a repo file, like those found in `/etc/yum.repos.d/`. This tells the Cachi2 where to find your rpm and its dependencies.
+
+. Copy any necessary yum/dnf repo files into your git repository. If you are using a fedora base image, that looks like:
+
++
+[source]
+----
+$ cp /etc/yum.repos.d/fedora.repo .
+----
+
+. Run the following command to resolve your `rpms.in.yaml` file and produce a `rpms.lock.yaml` file.
+
++
+[source]
+----
+$ rpm-lockfile-prototype --image YOUR_BASE_IMAGE rpms.in.yaml <1>
+----
+<1> The produced `rpms.lock.yaml` file will include only your requested dependency plus its transitive dependencies, minus any rpms that are already installed in the provided base image.
+
+. Go to the `.tekton` directory and find the `.yaml` files related to the `*pull request*` and `*push*` processes.
+. Configure the hermetic pipeline by adding the following parameters in both `.yaml` files:
+
++
+[source,yaml]
+----
+spec:
+    params:
+        -   ...
+        -   name: prefetch-input
+            value: '{"type": "rpm", "path": "."}' <1>
+----
+<1> The `*prefetch-input*` parameter specifies the path to the directory that has the lockfile. In this example, the `.` indicates that the rpm lockfile is in the repository root. Additionally, if you have multiple directories, you can provide the path to those directories in the JSON array format. For example, `[{"path": ".", "type": "rpm"}, {"path": "subpath/to/the/other/directory", "type": "rpm"}]`.
+
+. Additionally, pass an extra parameter to the `prefetch-dependencies` task to indicate that "dev package managers" should be enabled.
+
++
+[source,yaml]
+----
+spec:
+    tasks:
```

Comment:
```suggestion
spec:
    pipelineSpec:
        tasks:
```
Setting prefetch-input is done in the PipelineRun, while adding dev-package-managers is done in the Pipeline. This way, at least both the snippets look like a PipelineRun

Reply:
How about the change in a42dcb3a1d0aa6d9c737cc410ba1273abd37ee3f?

Reply:
That works :+1: